src-vue/src/views/DeveloperView.vue

```diff
@@ -207,7 +212,22 @@ export default defineComponent({
             await persistentStore.clear();
             // ...and save
             await persistentStore.save();
-        }
+        },
+        async forceInstallNorthstar() {
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+            let install_northstar_result = invoke("install_northstar_caller", { gamePath: game_install.game_path, northstarPackageName: ReleaseCanal.RELEASE });
+            await install_northstar_result
+                .then((message) => {
+                    console.log(message);
```

Comment:
So you should update installed version here (the variable to update is `installed_northstar_version`)

Reply:
Wouldn't the best approach be to just call `_get_northstar_version_number` from `store.ts`?

https://github.com/R2NorthstarTools/FlightCore/blob/d75ba3fe4b4b95b50490e0d8febdaf7d5a4c75ce/src-vue/src/plugins/store.ts#L453-L474

But as far as I can tell that's a private function?

Reply:
Yep, but you still can invoke it through the `checkNorthstarUpdates` store mutation:
```javascript
store.commit('checkNorthstarUpdates');
```

Reply:
Aight, fixed in ed315e25b2d9c5216f272b44224fc9c140332fdc